### DISCUSSION TITLE
Influx Data format - integer versus decimal

### DISCUSSION THREAD

**Discussion by raymop** [tagyoureit/nodejs-poolController#517] [App and O/S Config Questions] [answered]
After setting up the influx 1.8 interface, I queried the stored data for bodyTemps and ambientTemps and notice that the values are stored as integers, but are displayed in the dash panel with decimals.  Plus, when my grafana dashboard reads the data from influx, it only displays as integers.
Am I doing something wrong?  
Is there a config to set the # of decimals in influx?
Thanks,

**Comment by rstrouse** [comment, marked as answer]
These bindings were created at a time when the supported master panels treated temperature as integers.  Unfortunately, you cannot simply change the format after the Influx database is created and it will throw errors if you simply change the binding.   So if you pull njsPC I added the following bindings to the ambientTemps that are floating point values.
airTempF
solarTempF

On the bodyTemps it now includes a F value that indicates it is floating point.

**Comment by raymop** [comment]
Ah. Ok. 
I see what you’ve done. 
Terrific. I dropped my existing influx database and created a new one. That was easiest for me. 
And I see the F appended to the series. 
Thx.
